contrib/singularity/Singularity.1.0.0-0.1.4+

```diff
@@ -0,0 +1,36 @@
+BootStrap: debootstrap
```

Comment:
Feel free to skip reviewing this file. Its content is identical to Singularity. It is a copy of Singularity.

Reply:
Would a symlink work?

Reply:
No. Singularity Hub will try to find **ALL** files with the name prefix Singularity, and give it a tag according to the extension of the file names. Without any file name extension, the image would be automatically tagged as **latest**. For me, I want to keep evolving Singularity, which will has **latest** tag, and tag some of the other images. Their contents are different, so individual files are necessary.

However, I am thinking if a folder to host all Singularity files would be better from the maintaining point of view. We may refer to [Naming Recipes](https://github.com/singularityhub/singularityhub.github.io/wiki/Build-A-Container) session to design how we will host these recipes. This kind of change is recommended to be included in this PR because it may cause review difficulty and it is out of scope a bit.